protostar/configuration_file/configuration_file.py

```diff
@@ -86,6 +86,17 @@ def set_command_names_provider(
     ):
         pass
 
+    @staticmethod
+    def create_appending_cairo_path_suggestion() -> str:
+        return "\n".join(
+            [
+                "You may want to add your new library's path to the 'cairo_path' "
```

Comment:
```suggestion
                "You may want to add your new library's project root path to the 'cairo_path' "
```

It's not exactly the library's path. If that were a case, we wouldn't need this suggestion.